libs/designsystem/header/src/header.component.scss

```
@@ -141,6 +141,12 @@ $margin-inline: utils.size('s');
   margin-bottom: utils.size('xxxs');
 }
 
+.flag .custom-flag-section {
+  display: flex;
+  align-items: center;
+  min-height: utils.line-height('l');
+}
```

Consider simplifying this, as the current implementation only gives the expected height for the default text size (1rem/16px). For larger flag text sizes the spacing between flag and title will be smaller than expected, unless we define the margin between the two elements like this, instead of using flexbox etc.

```suggestion
.flag .custom-flag{
  margin-block: 4px; // use size variable here
}
```